client/src/components/card/effect-card-svg.module.css

```diff
@@ -23,6 +23,10 @@
 	fill: rgb(226, 202, 139);
 }
 
+.rarity {
```

Comment:
Can you please rebase on latst dev so you don't have the star image change in another commit?